libs/blocks/global-navigation/utilities/menu/menu.js

```diff
@@ -173,6 +173,18 @@ const decoratePromo = (elem, index) => {
     elem.classList.add('feds-promo--dark');
   }
 
+  if (!isImageOnly) {
+    const imageElemMaybe = imageElem && elem.removeChild(elem.querySelector('.feds-promo-image'));
+    const innerContainer = document.createElement('div');
+    innerContainer.classList.add('feds-promo-inner-container');
```

Comment:
For consistency, we could simply use the  `toFragment` method

Reply:
Changed 👍